app/configuration/Config.scala

```diff
@@ -111,7 +114,7 @@ object Config {
         prpIds.digitalPackYearly
       )),
       campaignName = "DigiPack - £30 digital gift card",
-      codes = PromoCodeSet(PromoCode("DGA88"), PromoCode("DGB88")),
+      codes = PromoCodeSet(PromoCode("DGA88"), promoCodes:_*),
```

Comment:
One could rewrite this like:
val promoCodes = (88 to 94).foldLeft(Seq(jellyFishPromoCode))((seq, i) =>  seq :+ PromoCode(s"DGA$i") :+ PromoCode(s"DGB$i"))